src/backend/api/Fusion.Resources.Api/Fusion.Resources.Api.csproj

```diff
@@ -14,9 +14,9 @@
     <PackageReference Include="FluentValidation.AspNetCore" Version="8.6.2" />
     
     <PackageReference Include="Bogus" Version="29.0.1" />
-    <PackageReference Include="Fusion.Integration" Version="3.1.0-preview-7" />
-    <PackageReference Include="Fusion.Integration.Authorization" Version="3.1.0-preview-7" />
-    <PackageReference Include="Fusion.Integration.Org" Version="3.1.0-preview-7" />
+    <PackageReference Include="Fusion.Integration" Version="3.1.0-local-2" />
```

Comment:
I have reserved local-1 through 4 on my computer ;)

Reply:
Ok, will do local-100 as my starting point, just in case :D